scss/_patterns_grid.scss

```diff
@@ -180,3 +157,204 @@
     }
   }
 }
+
+@mixin vf-l-fluid-breakout(
```

Comment:
As this is a separate mixin and a layout outside of the grid it should be in a separate file called `_layouts_something`.

Reply:
done